apps/admin/src/env.mjs

```diff
@@ -22,6 +22,9 @@ export const env = createEnv({
         ADMIN_MAIL: z.string().email().optional(),
 
         BLOB_READ_WRITE_TOKEN: z.string().optional(),
+
+        ML_API_URL: z.string(),
+        ML_API_KEY: z.string(),
```

Comment:
Bitte in den env.samples ergänzen.
Macht es Sinn, für diesen API-Endpunkt einen eigenen Key zu nutzen?

Reply:
Ja, weil das ein externer Service ist. Für die Kommunikation zwischen Front- und Backend kann der normale Key verwendet werden.